generated/nidaqmx/nidaqmx.proto

```diff
@@ -2948,6 +2953,1884 @@ enum CurrentUnits2 {
   CURRENT_UNITS2_FROM_CUSTOM_SCALE = 10065;
 }
 
+enum DAQmxErrors {
+  DA_QMX_ERRORS_UNSPECIFIED = 0;
+  DA_QMX_ERRORS_REMOTE_SENSE = -209888;
+  DA_QMX_ERRORS_OVER_TEMPERATURE_PROTECTION_ACTIVATED = -209887;
+  DA_QMX_ERRORS_MULTI_TASK_CFG_SAMP_RATE_NOT_SUPPORTED_WITH_PROP_SET = -209886;
+  DA_QMX_ERRORS_MULTI_TASK_CFG_SAMP_RATE_CONFLICTING_PROP = -209885;
+  DA_QMX_ERRORS_NO_COMMON_SAMP_RATE_FOUND_NO_REPEAT_SAMPS = -209884;
+  DA_QMX_ERRORS_NO_COMMON_SAMP_RATE_FOUND = -209883;
+  DA_QMX_ERRORS_MULTI_TASK_CFG_DOES_NOT_SUPPORT_MULTI_DEV_TASK = -209882;
+  DA_QMX_ERRORS_MULTI_TASK_SAMP_RATE_CFG_NOT_SUPPORTED = -209881;
+  DA_QMX_ERRORS_DEBUG_SESSION_NOT_ALLOWED_TIMING_SOURCE_REGISTERED = -209880;
+  DA_QMX_ERRORS_DEBUG_SESSION_NOT_ALLOWED_WHEN_LOGGING = -209879;
```

Comment:
We don't need the error/warning enums in `nidaqmx.proto`.

Also, the enum values are converted to snake_case incorrectly: `DA_QMX_...`

We may need a special case in the grpc-device codegen to exclude these.

Reply:
Added a validation suppression rule to exclude these. 

https://github.com/ni/grpc-device/pull/881/commits/b23fc4a4e0cdcaaf477b37b8eebe1af046b5df87